test/minbb.cc

```diff
@@ -0,0 +1,97 @@
+#include "test.h"
```

Comment:
here we test the minimum bounding box functionality by first configuring graphreader to nothing and seeing its an invalid bb and then configure it to utrecht. we test both expansion and contraction.